neo/sys/platform.h

```diff
@@ -38,11 +38,9 @@ If you have questions concerning this license or the applicable additional terms
 
 // NOTE: By default Win32 uses a 1MB stack. Doom3 1.3.1 uses 4MB (probably set after compiling with EDITBIN /STACK
 // dhewm3 now uses a 8MB stack, set with a linker flag in CMakeLists.txt (/STACK:8388608 for MSVC, -Wl,--stack,8388608 for mingw)
-// Linux has a 8MB stack by default, and so does macOS, at least for the main thread.
-// Anyway, a 1MB limit for _alloca() should be safe even when using it multiple times
-// in the same function or callstack.
-// If there's a risk of bigger stack allocations, Mem_MallocA() should be used instead.
-#define ID_MAX_ALLOCA_SIZE 1048576 // 1MB
+// Linux has a 8MB stack by default, and so does macOS, at least for the main thread
+// anyway, a 2MB limit alloca should be safe even when using it multiple times in the same function
+#define ID_MAX_ALLOCA_SIZE 2097152 // 2MB
```

Comment:
is there a specific reason you increased this to 2MB?

Reply:
i dont remember changing that ? maybe something from TDM carried over :S

Reply:
no just checked this flag does not exist there and neither anywhere else...
no idea tbh.